hydra-cluster/test/Test/EndToEndSpec.hs

```diff
@@ -313,11 +367,50 @@ spec = around showLogsOnFailure $ do
                     metrics <- getMetrics n1
                     metrics `shouldSatisfy` ("hydra_head_events" `BS.isInfixOf`)
 
-    describe "hydra-node executable" $
+    describe "hydra-node executable" $ do
       it "display proper semantic version given it is passed --version argument" $ \_ ->
         failAfter 5 $ do
           version <- readCreateProcess (proc "hydra-node" ["--version"]) ""
           version `shouldSatisfy` (=~ ("[0-9]+\\.[0-9]+\\.[0-9]+(-[a-zA-Z0-9]+)?" :: String))
+      fit "hydra-node logs it's command line arguments" $ \_ -> do
+        failAfter 60 $
+          withTempDir "temp-dir-to-check-hydra-logs" $ \dir -> do
+            let hydraSK = dir </> "hydra.sk"
+            hydraSKey :: SigningKey HydraKey <- generate arbitrary
+            void $ writeFileTextEnvelope hydraSK Nothing hydraSKey
+            bracket
```

Comment:
There is a slight inconvenience here: We call hydra-node process which prints out the cmd line arguments and then fails with `userError` because of missing cardano vkeys. It would ideally just print and then we kill the process so we don't see any print coming out of the error. Note to self: find a way to _interrupt_ the process after second or two.

Reply:
I don't see the problem that the process dies before after printing the run options. If it doesn't, exiting the bracket will kill it.

You can use `withCreateProcess` here instead of the bracket.

Reply:
I though it might produce SIGTERM/KILL up on CI maybe so that's why I mentioned it